ppocr/utils/loggers/wandb_logger.py

```diff
@@ -0,0 +1,79 @@
+import os
+from .base_logger import BaseLogger
+
+class WandbLogger(BaseLogger):
+    def __init__(self, 
+        project=None, 
+        name=None, 
+        id=None, 
+        entity=None, 
+        save_dir=None, 
+        config=None,
+        **kwargs):
+        try:
+            import wandb
+            self.wandb = wandb
+        except ModuleNotFoundError:
+            raise ModuleNotFoundError(
+                "wandb is not installed."
```

Comment:
I think this text is not required. ModuleNotFound will indicate that this library is not present.